### DISCUSSION TITLE
Splitter_port: record simultaneously into 2 files with 2 config options

### DISCUSSION THREAD

**Discussion by matmicro** [raspberrypi/picamera2#598] [Q&A]
Is PiCamera2 providing a splitter_port option as it was available from PiCamera 1.xx ?

The goal is to be able to record simultaneously into 2 files: one file with quality.low resolution and a second file with quality.high resolution.

```
camera.start_recording(path, splitter_port=1, format=vdo_format, quality=vdo_quality)
camera.start_recording(path_sd, splitter_port=2,  format=vdo_format, quality=vdo_qualitySD, resize=vdo_resolutionSD)
```

**Comment by davidplowman** [comment]
Hi, thanks for the question.

There won't be anything like a "splitter port" but we are looking at adding a "multiple encode" feature which would be equivalent.

**Comment by lowflyerUK** [comment]
Hi!

[https://github.com/raspberrypi/picamera2/pull/551/commits/1463c23d1e98a39e1724a4ff786b579d0e1265ea](url) worked brilliantly for me! Looks like "multiple encode" is already there - it's just not quite so easy to set up the second encoder.

Thanks @davidplowman

**Comment by davidplowman** [comment]
There is actually another PR in progress that will make this easier. See https://github.com/raspberrypi/picamera2/pull/588.

**Reply by lowflyerUK** [reply]
Excellent!! Many thanks!
